templates/Project/project_details.html.twig

```diff
@@ -34,10 +34,10 @@
     </div>
     <div class="d-none d-lg-block col-lg-4 mt-2">
       {% include 'components/redirect_button.twig' with {
-        'id': 'projectCodeViewButton',
-        'url': path('code_view', {'id': project.id }),
+        'id': 'projectStealButton',
+        'url': path('projectStealButton', {'id': project.id }),
         'icon': 'extension',
-        'text': "codeview.title"|trans({}, "catroweb")
+        'text': "Steal"
```

Comment:
Since you have not used the translation system, just a heads up for the future:

`codeview.title` is registered by the developer in the `translations/catroweb.en.yaml` file as 
```
codeview:
    title: Steal
```

All other translation files; E.g `catroweb.de.yml` are translated in its own process (automated by GitHub Actions) using Crowdin -> More details here: https://catrobat.atlassian.net/wiki/spaces/CATWEB/pages/74383377/Crowdin+-+Translation+Management